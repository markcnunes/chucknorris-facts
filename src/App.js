import React from "react";
import Message from "./components/Message";
// import logo from './logo.svg';
import "./style/App.css";

function App() {
  return (
    <div className="App">
      <Message />
      {/* <a
          className="App-link"
          href="https://reactjs.org"
          target="_blank"
          rel="noopener noreferrer"
        >
          Learn React
        </a> */}
    </div>
  );
}

export default App;
