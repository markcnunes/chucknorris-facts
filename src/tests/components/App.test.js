import React from "react";
import { shallow } from "enzyme";
import App from "../../App";

describe("App test", () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<App />);
  });
  test("render component", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
