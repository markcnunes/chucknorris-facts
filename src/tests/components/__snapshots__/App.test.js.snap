// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App test render component 1`] = `
<div
  className="App"
>
  <Message />
</div>
`;
